src/TimelineEvent/index.js

```diff
@@ -79,6 +80,10 @@ TimelineEvent.propTypes = {
   /**
    * Timeline event content (any JSX)
    */
+  tooltip: PropTypes.string,
+  /**
+   * Hover tooltip content for the icon
+   */
```

Comment:
jsdoc should be above proptype

```suggestion
  /**
   * Hover tooltip content for the icon
   */
  tooltip: PropTypes.string,
```

Reply:
@akdetrick Updated!